unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Interfaces/IParcelScene.cs

```diff
@@ -38,5 +39,6 @@ public interface IParcelScene
         void SetEntityParent(long entityId, long parentId);
         void RemoveEntity(long id, bool removeImmediatelyFromEntitiesList = true);
         bool IsInitMessageDone();
+        Bounds GetOuterBounds();
```

Comment:
related to previous comment about not being tied up to the scene instance bounds

Reply:
We won't be able to get rid of that method ( or exposing the outerBounds property itself but that has other problems that I couldn't get around ), since the outer bounds are calculated internally during the scene setup and later just used.

Reply:
Yes but we have the same problem as the previous point, if I do that, we are having more stuff duplicated, as the SDK6 will still be having the internal outer bounds calculation on its scene setup...

Reply:
I'll try to move that to an external scenes dictionary

Reply:
Check now